modules/services/featureflag/src/main/java/au/com/shiftyjelly/pocketcasts/featureflag/providers/FirebaseRemoteFeatureProvider.kt

```diff
@@ -15,8 +15,12 @@ class FirebaseRemoteFeatureProvider @Inject constructor(
     override fun isEnabled(feature: Feature): Boolean =
         firebaseRemoteConfig.getBoolean(feature.key)
```

Comment:
If for some reason this key does not exist in Firebase, this `getBoolean` method will fall back to the default value we give to Firebase when we call `setDefaultsAsync`. We're not doing that currently, which means it would always default to `false`. I'm thinking maybe we should add an entry to our [FirebaseConfig](https://github.com/Automattic/pocket-casts-android/blob/main/modules/services/utils/src/main/java/au/com/shiftyjelly/pocketcasts/utils/config/FirebaseConfig.kt) for this key so it has a default value.

Ideally we would make that happen automatically based on the `Feature` class, but that's a larger change, so perhaps for this PR we just hardcode the key and default value into `FirebaseConfig`?

I made a separate PR targeting `main` (https://github.com/Automattic/pocket-casts-android/pull/1441) that I think helps make it so that when we create a new feature flag it will get the appropriate default value set automatically.

Reply:
It's better to add a default value to `RemoteConfig` to handle a scenario when the key doesn't exist in `Firebase`. 

> perhaps for this PR we just hardcode the key and default value into FirebaseConfig

Added in https://github.com/Automattic/pocket-casts-android/pull/1437/commits/68d7e94d69429472f9ef68ba9493c9f79a7efa65

> I made a separate PR targeting main (https://github.com/Automattic/pocket-casts-android/pull/1441) that I think helps make it so that when we create a new feature flag it will get the appropriate default value set automatically.

That's so nice! I'll shortly take a look.

Reply:
> It's better to add a default value to RemoteConfig to handle a scenario when the key doesn't exist in Firebase.

Can you tell me a bit more about what you mean here?

Reply:
Oh, I just meant to agree with you to add a default value to [FirebaseConfig](https://github.com/Automattic/pocket-casts-android/blob/main/modules/services/utils/src/main/java/au/com/shiftyjelly/pocketcasts/utils/config/FirebaseConfig.kt).